### PR TITLE
Reorganize some callouts in etcd admin task

### PR DESCRIPTION
**Pull request by sftim**
Improve https://kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/

- Aim to reduce the large number of note callouts
  - promote one to a caution.

[Preview](https://deploy-preview-46777--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/configure-upgrade-etcd/)

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -333,6 +333,20 @@ for information on how to add members into an existing cluster.
...
+restore instances of etcd. Instead, follow these steps to restore etcd:
+
+- stop *all* API server instances
+- restore state in all etcd instances
+- restart all API server instances
+
+The Kubernetes project also recommends restarting Kubernetes components (`kube-scheduler`,
+`kube-controller-manager`, `kubelet`) to ensure that they don't rely on some
+stale data. In practice the restore takes a bit of time.  During the
+restoration, critical components will lose leader lock and restart themselves.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:347]: Text is moved, not new

**Review by dipesh-rawat — COMMENTED** [review]
The proposed tweaks are good!

/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
/approve

**Pull request merged**